index.php

```diff
@@ -42,6 +42,13 @@
     ]
 ];
 
+function price_modify($price) {
+    ceil($price);
+    if ($price>1000){
```

Comment:
Не теряй пробелы, что бы код не слипся. 
```suggestion
    if ($price > 1000) {
```